docs/config/backend.adoc

```diff
@@ -1,20 +1,20 @@
 = Backend
 
-A modul konfigurációja MicroProfile Config-on keresztül történik, ami által több módon is lehetséges a szükséges értékek megadása.
+The module's configuration is managed via MicroProfile Config, which allows specifying necessary values in multiple ways.
 
-A MicroProfile Config képes egy megadott kulcsot kikeresni az összes elérhető forrásból és a legmagasabb prioritású értéket használja fel.
+MicroProfile Config can retrieve a given key from all available sources and uses the highest priority value.
 
-Alap konfiguráció a project-defaults.yml-en keresztül kerül megadásra, melyet bővíteni lehet, és környezetenként változhat, de nem kell minden értéket megadni, csak ami változik az alap beállításokhoz képest.
+The base configuration is provided via `project-defaults.yml`, which can be extended and may vary per environment. It's not necessary to specify every value; only those that differ from the default settings.
 
-*Lehetséges módok prioritási sorrendben:*
+*Possible modes in order of priority:*
 
-* Rendszer változók
-* Környezeti változók
+* System variables
+* Environment variables
 
 ifdef::generatedDoc[]
 
 == DOOKUG-DOCUMENT service
 
 include::../generated/dookug-api-common-config.adoc[leveloffset=+1]
```

Comment:
a @ConfigDoc leírásokat is érdemes lehet lefordítani szerintem amik ide generálódnak

Reply:
jaja, csak azokat kulon MRben gondoltam

Reply:
vegul beletettem most :)